controllers/default_deny_network_policy.go

```diff
@@ -95,6 +115,27 @@ func (r *DefaultDenyNetworkPolicyReconciler) Reconcile(ctx context.Context, name
 		xdsPort := intstr.FromInt(15012)
 		networkPolicy.Spec.Egress[2].Ports[0].Port = &xdsPort
 
+		// Egress rule for instana-agents - TODO: find way to specify CIDR block from ConfigMap
+		networkPolicy.Spec.Egress[3].To = make([]networkingv1.NetworkPolicyPeer, 1)
+		networkPolicy.Spec.Egress[3].To[0].IPBlock = &networkingv1.IPBlock{}
+		networkPolicy.Spec.Egress[3].To[0].IPBlock.CIDR = instanaConfigMap.Data["cidrBlock"]
+
+		startPort := 32768
+		endPort := 60999
+		amountOfPorts := (endPort - startPort) + 1
+
+		networkPolicy.Spec.Egress[3].Ports = make([]networkingv1.NetworkPolicyPort, amountOfPorts)
+		instanaProtocol := corev1.ProtocolTCP
+		// specify ephemeral ports that may or may not be used by the JVMs to communicate with instana-agent
+		// - yes, this will result in a horribly long list, but NetworkPolicy doesn't support portRange
+		// until Kubernetes v1.25, while Anthos is currently on 1.24
+
+		for i := 0; i < amountOfPorts; i++ {
+			jvmPort := intstr.FromInt(startPort + i)
```

Comment:
I mitt hode er dette ryddigere, hva tenker du om det?

```suggestion
		for i := startPort; i <= endPort; i++ {
			jvmPort := intstr.FromInt(i)
```

Reply:
Må isåfall ha en annen iterator for networkPolicy.Spec.Egress[3].Ports[i] da, siden i da ikke starter på 0 ?

Reply:
Good point, jeg missa at man bruker i i linjene under. Da er det veldig forståelig at du gjorde det sånn :ok_hand: